src/components/approval-type/ApprovalType.vue

```diff
@@ -237,20 +235,6 @@ export default class ApprovalType extends Vue {
     this.validate()
     return applicationDate
   }
-
-  /* Set court order number to empty if approved by registrar option is selected. */
- @Watch('courtOrderNumberText')
-  private setCourtOrderNumber () {
-    // When going from valid to invalid (boundaries), the form's validity doesn't update immediately.
```

Comment:
@JazzarKarim Do I need to test any specific scenario to check this?

Reply:
Yes Vysakh. Can you please test the following scenarios:
- When going from a court order length of 4 to 5 and vice versa.
- When going from court order length of 20 to 21 and vice versa.

The reason is that the component sometimes doesn't register the immediately the correct invalid state of the form when going from a valid state. Please let me know if you are able to test these scenarios.

Reply:
Marking this conversation as resolved after our call 👍